rclcpp_components/src/component_manager.cpp

```diff
@@ -37,6 +37,9 @@ ComponentManager::ComponentManager(
 : Node(std::move(node_name), node_options),
   executor_(executor)
 {
+  declare_parameter("use_dedicated_thread", use_dedicated_thread_);
```

Comment:
@fujitatomoya do you mean that we could use `request->extra_arguments` to choose `Conatiner's Executor` or `Dedicated SingleThreadedExecutor`  that node should be loaded on at runtime? so we needn't global parameter `use_dedicated_thread`.

Reply:
i don't consider to change `use_dedicated_thread` at runtime,  the parameter `use_dedicated_thread` is only  initialized at constructor, which won't change at runtime. i don't understand the user case you mentioned.

Reply:
I was thinking both cases make sense,

- `use_dedicated_thread` is constructor option. once this option is chosen, any nodes that are loaded will have dedicated `SingleThreadedExecutor`.
- `use_dedicated_thread` parameter, so that user can have full control either loading nodes on dedicated `SingleThreadedExecutor` or the executor that ComponentManager spins on. (control where to load the nodes.)

I would like to hear from other opinion.